quaistats/quaistats.go

```diff
@@ -712,6 +713,12 @@ func (s *Service) assembleBlockStats(block *types.Block) *blockStats {
 		log.Error("error computing tps", "error", err.Error())
 	}
 
+	if appendTime, ok := block.GetAppendTime(); ok {
+		ret.AppendTime = appendTime
+	} else {
+		log.Error("error getting append time")
```

Comment:
So if we failed to look up the append time, we report 0ms? I would rather report something obviously wrong, like -1 or U64_MAX

I'm realizing this is an issue with the TPS report above too.